Sources/Commands/SwiftTool.swift

```diff
@@ -794,7 +794,11 @@ public class SwiftTool {
             // can be used to build for any Apple platform and it has it's own
             // conventions for build subpaths based on platforms.
             let dataPath = buildPath.appending(
+<<<<<<< HEAD
                 component: options.buildSystem == .xcode ? "apple" : triple.tripleString)
+=======
+                component: options.buildSystem == .xcode ? "apple" : triple.platformBuildPathComponent())
+>>>>>>> eb54bad4 (Disuse unversioned triples on non-Darwin platforms.)
```

Comment:
@3405691582, git conflict needs to be reconciled.

Reply:
ugh, sorry.